main.js

```diff
@@ -95,7 +107,62 @@ const profileOnDom = (array) => {
     `;
   });
 
-  renderToDom("#sidebar", domString)
+  renderToDom("#sidebar", domString);
+};
 
+//Packages to DOM
+const allPackages = (array) => {
+  let domString = "";
+  for (const package of array) {
+    domString += `<div class="card card-package" style="width: 18rem;">
+    <div class="card-body">
+      <h5 class="card-title">${package.title}</h5>
+      <p class="card-text">${package.description}</p>
+      <button type="button" class="btn btn-secondary btn-sm">Learn More</button>     
+    </div>
+  </div>`;
+  }
+  renderToDom("#packages", domString);
 };
-// profileOnDom(profiles)
+
+//Create Package Form
+const packageForm = () => {
+  const domString = `
+  <form id="inputFormPackage">
+    <h3 class="form-header">Create a new project</h3>
+    <div class="form-floating mb-3">
+      <input class="form-control form-control-lg" type="text" placeholder="Example 1" id="projectName" aria-label="projectName" required>
+      <label for="projectName">Project Board Name</label>
+    </div>
+    <div class="mb-3">
+    <label for="exampleFormControlTextarea1" class="form-label">Description</label>
+    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+    <button type="submit" class="btn btn-success" id="form-submit">Create Package</button>
+  </form>`;
+  renderToDom("#packages-form", domString);
+
+  const form = document.querySelector("#packages-form");
+  form.addEventListener("submit", (e) => {
+    e.preventDefault();
+
+    const newPackageObj = {
+      id: packages.length + 1,
+      name: document.querySelector("#projectName").value,
+      description: document.querySelector("#exampleFormControlTextarea1").value,
+    };
+    packages.push(newPackageObj);
+    allPackages(packages);
+    form.reset();
+  });
+};
+
```

Comment:
Add an evenListener for .nav on line160 and make sure to put e.preventDefault(); in the event Listener as it constantly reloads the page. Then add an if statement that checks if an id is present in the packages in the navbar. (There is not currently an id but I will make a comment to add an id) After you check if the id is present, it should call allpackages and packageForm. Should look something like this:
![image](https://github.com/nss-evening-cohort-26/gitsub-untouchables/assets/153968439/3614d0c3-cd09-49bb-bb3c-dd78d280d421)